pkg/azuredx/resource_handler.go

```diff
@@ -141,8 +141,8 @@ func (adx *AzureDataExplorer) getSchema(rw http.ResponseWriter, req *http.Reques
 		QuerySource: "schema",
 	}
 
-	headers := map[string]string{}
-	response, err := adx.client.KustoRequest(req.Context(), cluster.ClusterUri, ManagementApiPath, payload, headers)
+	// Default to not sending the user request headers for schema requests
```

Comment:
I'm not sure what this comment means. Are we just saying that we are not sending the user request header because it's a schema request? i think im being dense 😆

Reply:
Exactly that, we don't need to send the header on every request. IIRC it should only be used for queries